packages/patch/src/__tests__/utils.ts

```diff
@@ -0,0 +1,21 @@
+import { buildSchema, GraphQLSchema, lexicographicSortSchema, parse, print } from 'graphql';
+import { Change, diff } from '@graphql-inspector/core';
+import { printSchemaWithDirectives } from '@graphql-tools/utils';
+import { patchSchema } from '../index.js';
+
+function printSortedSchema(schema: GraphQLSchema) {
+  return printSchemaWithDirectives(lexicographicSortSchema(schema));
+}
+
+export async function expectPatchToMatch(before: string, after: string): Promise<Change<any>[]> {
```

Comment:
Very nice helper!

Reply:
I think the name could be a bit more explicit? Maybe, just writing this because from looking at the first test it was not obvious to me what the helper does.

```
expectdiffAndApplyToMatch
```